tutorials/algorithms/01_algorithms_introduction.ipynb

```diff
@@ -15,7 +15,7 @@
    "source": [
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    from qiskit.algorithms.minimum_eigensolvers import VQE
</pre><p>I'd move this import to where <code>VQE</code> was moved in this PR, so the user can keep track on when they need to import. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Qiskit/qiskit-tutorials/pull/1370/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
I'd rather have all imports grouped -- since the cell is very short it's probably clear enough when VQE is required 🙂